src/atomistics/referencedata/wolframalpha.py

```
@@ -17,7 +18,7 @@ def _get_content_from_url(url: str) -> pandas.DataFrame:
     Returns:
         pandas.DataFrame: The content retrieved from the URL as a pandas DataFrame.
     """
-    content = pandas.read_html(requests.get(url).text)
+    content = pandas.read_html(StringIO(requests.get(url).text))
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)